doc/articles/contributing/guidelines/code-style.md

```diff
@@ -32,19 +39,45 @@ if (condition)
 }
 else
 {
-    // use braces even for single line conditions
+    // use braces even for single line conditions to prevent "goto fail"
+    // https://www.imperialviolet.org/2014/02/22/applebug.html
 }
 ```            
 
+### Filenames
+
+As defined over at https://github.com/unoplatform/uno/blob/master/src/PlatformItemGroups.props
+
+- ClassName.wasm.cs
+- ClassName.iOS.cs
+- ClassName.macOS.cs
+- ClassName.iOSmacOS.cs
+- ClassName.Android.cs
+- ClassName.Xamarin.cs
+- ClassName.UWP.cs
+- ClassName.net.cs
+- ClassName.netstd.cs
+
+### Strings
+
+- Always consider globalization thus usage of `InvariantCulture`
+- Never concatenate strings ie. `"string a" + " " + "string b"` without good cause.
+- Prefer [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) over [stringbuilder](https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder)
+
+```csharp
+double speedOfLight = 299792.458;
+string messageInInvariantCulture = FormattableString.Invariant($"The speed of light is {speedOfLight:N3} km/s.");
+```
+
 ## Integration Tests
 
 ```csharp
-[ActivePlatforms(Platform.Android, Platform.Browser, Platform.iOS)]
 [TestFixture]
 public class LocalSettings_Tests : SampleControlUITestBase
 {
     [Test]
     [AutoRetry]
+    [ActivePlatforms(Platform.Android, Platform.Browser, Platform.iOS)]
```

Comment:
This must not be the default. Tests should run on all platforms by default, even those not listed yet.